src/hal/utils/halcmd_main.c

```diff
@@ -279,10 +281,18 @@ int main(int argc, char **argv)
     {
 	char cmdline[200];
```

Comment:
We should avoid magic numbers. Can you please define `MAX_CMD_LENGTH` somewhere.